### ISSUE TITLE
Wrong histogram buckets?

### ISSUE THREAD

**Issue by Allineer** [VictoriaMetrics/metrics#8]
example:

```go
package main

import (
	"fmt"
	"github.com/VictoriaMetrics/metrics"
)

func main() {
	var bb bytes.Buffer
	test := metrics.NewHistogram(`test`)
	test.Update(1000)
	test.Update(1001)
	test.Update(1002)
	metrics.WritePrometheus(&bb, false)
	fmt.Printf("%s", bb)
```

result:

```go
test_bucket{vmrange="9.5e2...1.0e3"} 3
test_sum 3003
test_count 3
```

expected:

```go
test_bucket{vmrange="9.5e2...1.0e3"} 1
test_bucket{vmrange="1.0e3...1.5e3"} 2
test_sum 3003
test_count 3
```

**Comment by valyala** [comment]
This looks like a bug. I'll try to resolve it soon.

**Comment by valyala** [comment]
The bug should be fixed in the commit https://github.com/VictoriaMetrics/metrics/commit/d81dd5ec9c3a8549c72b4b389ab2b2aff8fbb208 . This commit is included in the release `v1.11.3`.

**Comment by Allineer** [comment]
Спасибо, Саша!
